### PR TITLE
fix: update README to include an inbox example, ordering inbox above chat

### PR DESCRIPTION
**Pull request by kevindingens**
https://www.notion.so/change-default-examples-in-react-README-3b49189a9b3d4128851a30df07396a79

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,31 @@ The component above is a self-contained button that opens a notifications modal
...
+
+return <Inbox 
+  wrapperClassName="p-2 h-full overflow-hidden rounded-2xl shadow-2xl shadow-neutral-800 border border-neutral-600" 
+  wallet={wallet} 
+/>;
+
+// ...
+```
+
+The component above contains a wallet's current inbox of current chats/notifications. From this inbox, a user can create, browse, and send chats of their own. Props are outlined as follows.
```
Comment (kevindingens) [inline README.md:86]: From what I know playing around with inbox example, this is true, but need a check

Comment (fsher) [reply README.md:86]: This is correct, yes!

I'd rephrase a bit, the "send chats" part bothers me a bit, sorry :D Should we do something like: `create, browse chats, and send messages.` Wdyt?

**Review by fsher — APPROVED** [review]
Well done! Left a minor comment. May be wrong there, feel free to address it!

File: README.md (hunk 1 above)
Comment (kevindingens) [reply README.md:86]: Agreed. Updating now
